demos/appyx-interactions/web/src/jsMain/kotlin/com/bumble/appyx/interactions/main.js.kt

```diff
@@ -0,0 +1,77 @@
+package com.bumble.appyx.interactions
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.Surface
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.onSizeChanged
+import androidx.compose.ui.unit.IntSize
+import androidx.compose.ui.window.Window
+import com.bumble.appyx.components.internal.testdrive.TestDriveExperiment
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_amber_500
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_blue_500
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_blue_grey_500
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_cyan_500
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_grey_500
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_indigo_500
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_light_blue_500
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_light_green_500
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_lime_500
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_pink_500
+import com.bumble.appyx.transitionmodel.testdrive.ui.md_teal_500
```

Comment:
These colors are from material design, maybe they could live in common rather than testdrive

Reply:
They are defined in `appyx-components:internal:common` module. Shall I keep them in same module but define them out of the `testdrive` package?